### ISSUE TITLE
PHP Fatal error:  Cannot redeclare from() 

### ISSUE THREAD

**Issue by umpirsky** [Athari/YaLinqo#8]
I tried to use https://github.com/hoaproject/Ruler and https://github.com/Athari/YaLinqo in same project.

composer install failed with:

```
PHP Fatal error:  Cannot redeclare from() (previously declared in vendor/hoa/core/Consistency.php:890) in vendor/athari/yalinqo/YaLinqo/Linq.php on line 21
```

There is obvious collision in `from()` function. [Hoa checks if function exists](https://github.com/hoaproject/Core/blob/master/Consistency.php#L883-L888), I guess YaLinqo should do the same.

**Comment by cursedcoder** [comment]
+1

**Comment by Athari** [comment]
I don't like that it makes the code more fragile... I'd rather move the function into namespace, but `use function` is available only in PHP 5.6+. :worried: But adding this check won't hurt I guess.
